zerver/lib/export.py

```diff
@@ -1461,6 +1462,8 @@ def import_uploads_local(import_dir: Path, processing_avatars: bool=False,
         for record in records:
             if record['s3_path'].endswith('.original'):
                 user_profile = get_user_profile_by_id(record['user_profile_id'])
+                if os.path.exists(medium_file_path):
+                    os.remove(medium_file_path)
```

Comment:
This doesn't work, because of how variable scoping works in Python; `medium_file_path` needs to be set in the same loop iteration where it's used.